src/QMCWaveFunctions/Jastrow/RadialJastrowBuilder.cpp

```diff
@@ -513,12 +599,26 @@ std::unique_ptr<WaveFunctionComponent> RadialJastrowBuilder::buildComponent(xmlN
     // it's a one body jastrow factor
     if (Jastfunction == "bspline")
     {
-      return createJ1<BsplineFunctor<RealType>>(cur);
+      if (SpinOpt.find("yes") < SpinOpt.size())
```

Comment:
This would be more straightforward as `if (SpinOpt == "yes")`
(Unless I'm missing something about the values of SpinOpt - it is just "yes" or "no"?)

Reply:
Thanks I agree it will be just yes or no. I will update this accordingly

Reply:
https://github.com/QMCPACK/qmcpack/blob/9dfd387e343ed0ecab8583a7de4c26baba225794/src/io/OhmmsData/AttributeSet.h#L58
Use the third argument to restrict input options.

Reply:
updated. Thanks didn't know about that

Reply:
I think the default should be no. So `{"no", "yes"}`. Don't do the change now. I'm putting up a lot of review comments. I think we we should take the chance to do our best as the old one is intended to be deleted.